server/image_worker.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python
```

Comment:
Maybe the contents of this file should be moved into docker_resource.py. That way all the caching stuff is in one file. Not sure if i would use the functions in this file for anything else.